Jenkinsfile

```diff
@@ -22,6 +22,7 @@ node('boardfarm') {
         sh 'echo "route add default gw 192.168.0.1" > /dev/ttyUSB0'
         sh 'sleep 10'
         sh 'echo "sed -i \'$ a nameserver 8.8.4.4\' /etc/resolv.conf" > /dev/ttyUSB0'
+        sh 'echo "sed -i \'/signature/d\' /etc/opkg.conf" > /dev/ttyUSB0'
```

Comment:
do the builds from CI contain signature and are expected to pass signature checks? I prebuilt an image with kmod support for USB adapter, hence we can't use the official image.